lib/srv/db/proxyserver.go

```diff
@@ -516,6 +516,16 @@ func (s *ProxyServer) Proxy(ctx context.Context, proxyCtx *common.ProxyContext,
 		return trace.Wrap(err)
 	}
 
+	var labels prometheus.Labels
+	if len(proxyCtx.Servers) > 0 {
+		labels = getLabelsFromDb(proxyCtx.Servers[0].GetDatabase())
```

Comment:
```suggestion
		labels = getLabelsFromDB(proxyCtx.Servers[0].GetDatabase())
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms